src/library/filescanner_ffmpeg.c

```diff
@@ -778,3 +802,234 @@ scan_metadata_ffmpeg(struct media_file_info *mfi, const char *file)
 
   return 0;
 }
+
+static int
+file_clone_metadata(AVFormatContext* in_fmt_ctx, const char* dest)
```

Comment:
Doesn't this function actually clone the entire file, not just the metadata?

Reply:
yes full clone.  renamed